plugins/GSdx/GSState.cpp

```diff
@@ -44,11 +44,11 @@ GSState::GSState()
 	, m_crc(0)
 	, m_options(0)
 	, m_frameskip(0)
+	, m_mipmap(1)
 {
 	// m_nativeres seems to be a hack. Unfortunately it impacts draw call number which make debug painful in the replayer.
 	// Let's keep it disabled to ease debug.
 	m_nativeres = theApp.GetConfigI("upscale_multiplier") == 1 || GLLoader::in_replayer;
-	m_mipmap = theApp.GetConfigI("mipmap");
```

Comment:
I don't think this file and GSRendererSW.cpp actually need to be changed. It'll also make the OpenCL renderer always start with mipmapping enabled.

Apart from that, the code seems fine.